components/NavMenu.js

```diff
@@ -0,0 +1,65 @@
+import Link from "next/link";
+import { useState } from "react";
+import styled from "styled-components";
+import { useRouter } from "next/router";
+
+export default function NavMenu() {
+  const { pathname } = useRouter();
+
+  const [hamburgerMenu, setHamburgerMenu] = useState(false);
+  const toggleHamburgerMenu = () => {
+    setHamburgerMenu(!hamburgerMenu);
+  };
+
+  return (
+    <NavWrapper>
+      <NavButton onClick={toggleHamburgerMenu}>MENU</NavButton>
+      <BurgerMenu hamburgerMenu={hamburgerMenu}>
```

Comment:
It is best practice to use conditional rendering instead of switching the display css property to none.

Reply:
added conditional rendering